src/components/NavigationComponents/Navigation/__tests__/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import NavigationContainer from '../index';
+
+describe('Navigation component', (): void => {
+  it('renders correctly', (): void => {
+    const isOpen = false;
```

Comment:
You can put this in a beforeEach and remove the duplicated code here and on line 22 and 23

Reply:
Hello @benhalverson thank you for reviewing my code this Is my very first pull request. I updated the snapshots, and used beforeEach to remove the duplicate code